tests/test_fluctuation_dissipation.py

```diff
@@ -62,7 +62,7 @@ def fluctuation_dissipation_KS(M, fluctuation_method):
     ],
 )
 @pytest.mark.parametrize("hydrodynamicRadius", [0.95, 1.12])
-@pytest.mark.parametrize("numberParticles", [1, 2, 3, 10])
+@pytest.mark.parametrize("numberParticles", [10])
```

Comment:
Introduce 1 and 2. Bugs arise at 1 because it tends to be a corner case.
If it fails at 2 it is easier to debug than at 10.